projects/ngx-vest-forms/src/lib/directives/form.directive.ts

```diff
@@ -243,9 +250,14 @@ export class FormDirective<T extends Record<string, any>>
    * This ensures that form controls are fully created and available before
    * attempting to establish validation dependencies. The effect() in the constructor
    * handles subsequent reactive updates when input signals change.
+   *
+   * NOTE: This is no longer needed as the effect() in the constructor handles both
+   * initial and subsequent changes reactively. However, we keep this method for
+   * backward compatibility in case it's being called by user code.
    */
   public ngAfterViewInit(): void {
-    this.setupValidationConfig();
+    // No longer needed - effect() in constructor handles this reactively
```

Comment:
we don't need the comments about improved code etc. Only JSDoc about what/why/how